src/Service.Tests/Authentication/Helpers/WebHostBuilderHelper.cs

```diff
@@ -75,8 +75,19 @@ public static async Task<IHost> CreateWebHost(
                             else
                             {
                                 EasyAuthType easyAuthProvider = (EasyAuthType)Enum.Parse(typeof(EasyAuthType), provider, ignoreCase: true);
-                                services.AddAuthentication()
-                                    .AddEasyAuthAuthentication(easyAuthProvider);
+
+                                // When EasyAuth provider is AppService, set the default scheme accordingly so
+                                // AppServiceAuthentication is registered and available during tests.
+                                if (easyAuthProvider == EasyAuthType.AppService)
+                                {
+                                    services.AddAuthentication(EasyAuthAuthenticationDefaults.APPSERVICEAUTHSCHEME)
+                                        .AddEasyAuthAuthentication(easyAuthProvider);
+                                }
+                                else
+                                {
+                                    services.AddAuthentication()
```

Comment:
how is the else part different here? what scheme gets registered when we dony specify any argument to `AddAuthentication`?